lib/operation_handler.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:fwupd/fwupd.dart';
+import 'package:meta/meta.dart';
+
+abstract class OperationHandler {
```

Comment:
The name "OperationHandler" doesn't seem to convey what this class is doing. Perhaps use "FwupdService" for this class and rename the existing service to "FwupdDbusService"?

Reply:
It might then also make sense to rename DryrunService to FwupdMockService.

Reply:
@BLKKKBVSIK any thoughts on the "OperationHandler" naming?